Voilà, j'ai mis à jour en suivant les commentaires de @firm1 et j'ai mis le texte de Coyote !

Par contre, je n'ai pas réussi à faire de TU. En lançant la commande `python manage.py test zds.forum.tests.tests.ForumGuestTests.test_old_post_limit` avec [ce code](http://pastebin.com/embed_iframe.php?i=d2eTYRWK), j'obtiens cette erreur :

``` bash
Creating test database for alias 'default'...
F
======================================================================
FAIL: test_old_post_limit (zds.forum.tests.tests.ForumGuestTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/situphen/Documents/Dev/Python/zdsenv/zds-site/zds/forum/tests/tests.py", line 1372, in test_old_post_limit
    self.assertEqual(topic.old_post_warning(), True)
AssertionError: False != True

----------------------------------------------------------------------
Ran 1 test in 0.107s

FAILED (failures=1)
Destroying test database for alias 'default'...
```

--- a/templates/misc/message_form.html
+++ b/templates/misc/message_form.html
@@ -27,6 +27,16 @@
         {% trans "Vous êtes seul dans cette conversation" %}.
     </div>
 {% else %}
+    {% if topic.old_post_warning %}
+        <div class="alert-box warning">
+            <p>
+                {% blocktrans with old_post_limit_days=app.forum.old_post_limit_days %}
+                    Ce sujet est sans activité depuis plus de {{ old_post_limit_days }} jours. Êtes-vous certain de vouloir y contribuer ?
+                {% endblocktrans %}
+            </p>
+        </div>
+    {% endif %}
+
     <section class="topic-message">
         <form action="{{ form_action }}" method="post">
             {% include "misc/message_user.html" with hide_metadata=True %}

--- a/zds/forum/models.py
+++ b/zds/forum/models.py
@@ -9,7 +9,7 @@ import string
 import uuid
 
 from django.contrib.auth.models import Group, User
-from datetime import datetime
+from datetime import datetime, timedelta
 from django.core.urlresolvers import reverse
 from django.utils.encoding import smart_text
 
@@ -347,6 +347,22 @@ class Topic(models.Model):
         if last_user_post and last_user_post == self.get_last_post():
             t = datetime.now() - last_user_post.pubdate
             if t.total_seconds() < settings.ZDS_APP['forum']['spam_limit_seconds']:
+                return True
+
+        return False
+
+    def old_post_warning(self):
+        """
+        Check if the last message was written a long time ago according to `ZDS_APP['forum']['old_post_limit_days']`
+        value.
+
+        :return: `True` if the post is old (users are warned), `False` otherwise.
+        """
+        last_post = self.last_message
+
+        if last_post is not None:
+            t = last_post.pubdate + timedelta(days=settings.ZDS_APP['forum']['old_post_limit_days'])
+            if t < datetime.today():
                 return True
 
         return False

--- a/zds/forum/tests/tests.py
+++ b/zds/forum/tests/tests.py
@@ -6,6 +6,8 @@ from django.test import TestCase
 from django.core.urlresolvers import reverse
 from zds.utils import slugify
 
+from datetime import datetime, timedelta
+
 from zds.forum.factories import CategoryFactory, ForumFactory, \
     TopicFactory, PostFactory, TagFactory
 from zds.member.factories import ProfileFactory, StaffProfileFactory
@@ -1357,6 +1359,19 @@ class ForumGuestTests(TestCase):
             topic_solved_sticky,
             get_topics(forum_pk=self.forum11.pk, is_sticky=True, filter='noanswer'),
         )
+
+    def test_old_post_limit(self):
+        topic = TopicFactory(forum=self.forum11, author=self.user, is_solved=False, is_sticky=False)
+
+        # Create a post published just now
+        PostFactory(topic=topic, author=self.user, position=1)
+        self.assertEqual(topic.old_post_warning(), False)
+
+        # Create a post published one day before old_post_limit_days
+        old_post = PostFactory(topic=topic, author=self.user, position=2)
+        old_post.pubdate = datetime.now() - timedelta(days=(settings.ZDS_APP['forum']['old_post_limit_days'] + 1))
+        old_post.save()
+        self.assertEqual(topic.old_post_warning(), True)
 
 
 def get_topics(forum_pk, is_sticky, filter=None):

--- a/zds/settings.py
+++ b/zds/settings.py
@@ -453,6 +453,7 @@ ZDS_APP = {
         'max_post_length': 1000000,
         'top_tag_max': 5,
         'home_number': 5,
+        'old_post_limit_days': 90
     },
     'topic': {
         'home_number': 6,

